libs/util/git/src/constants.ts

```diff
@@ -1,9 +1,7 @@
 export const accumulativePullRequestTitle =
   "chore(Amplication): Update Generated Code";
 
-export const accumulativePullRequestBody = `This pull request includes multiple commits by Amplication with the latest generated code.
-The pull request may also include commits from several resources and/or projects.
-You can safely merge this pull request to your default branch.
+export const accumulativePullRequestBody = `If you're looking to bring in a new data model, tweak existing ones, handle APIs, or spice things up with Amplication plugins, just swing by the Amplication platform at $link_to_service. Tell us what you need, and we'll handle all the coding magic for you.
```

Comment:
@Meyazhagan be careful as the requirements in the issue state to **add** that text and not replace it.